api/src/main/java/org/apache/iceberg/types/Type.java

```diff
@@ -112,10 +114,11 @@ Object writeReplace() throws ObjectStreamException {
   }
 
   abstract class NestedType implements Type {
+    @Override
     public boolean isNestedType() {
       return true;
     }
-
+    @Override
```

Comment:
There should be an empty line between the two methods.

Reply:
Thanks! Updated.